src/sentry/snuba/outcomes.py

```diff
@@ -460,3 +464,33 @@ def massage_outcomes_result(
         del result["intervals"]
     del result["query"]
     return result
+
+
+def run_metrics_outcomes_query(
+    query: QueryDict, organization, projects, environments
+) -> dict[str, list]:
+    start, end = get_date_range_from_params(query)
+    interval = parse_stats_period(query.get("interval", "1h"))
+
+    # TODO(metrics): add support for reason tag
+    plan = MetricsQueriesPlan().apply_formula("sum(c:metric_stats/volume@none) by (outcome.id)")
+    rows = run_metrics_queries_plan(
+        plan,
+        start=start,
+        end=end,
+        interval=int(3600 if interval is None else interval.total_seconds()),
+        organization=organization,
+        projects=projects,
+        environments=environments,
+        referrer="outcomes.timeseries",
```

Comment:
This is `outcomes.timeseries` so that it matches a referrer for other stas queries that use query builders. But we can definitely add it to the list of known referrers